compatibility/bazel_tools/daml_ledger/Sandbox.hs

```diff
@@ -143,8 +143,8 @@ nullDevice
     | otherwise =  "/dev/null"
 
 readPortFile :: Int -> String -> IO Int
-readPortFile 0 _file = do
-  T.hPutStrLn stderr "Port file was not written to in time."
+readPortFile 0 file = do
+  T.hPutStrLn stderr ("Port file was not written to '" ++ file ++ "' in time.")
```

Comment:
Is this output correct at all? It looks to me that an attempt is being made at reading a file, not writing it. I'm confused.

Reply:
Ok, I guess the idea here is that `readPortFile` happens in parallel with another process writing the file and the intention here is to report that.

Reply:
Correct, the Haskell side will start some service that is expected to write a port file and will wait for this to happen up to a timeout.